cli/studio.py

```diff
@@ -146,6 +146,7 @@ def _clean(scenario):
         "social_agents/*",
         "traffic/*",
         "history_mission.pkl",
```

Comment:
Since we are using `.shf` files now we can probably remove `"history_mission.pkl"` here

Reply:
Yeah, definitely we need to eventually, but I wanted to leave it there a little bit to get rid of any existing files on the next clean.

Reply:
Cool! One other thing we can remove is the `ijson` requirement in dependency. line 64 and 65 in setup.py and line 48 in requirements.txt. ijson was only used for imitation learning

Reply:
funny, I did, but I had to add it back.  It's still being used in `genhistories.py` to convert old json files.